tools/register-arns-ants.js

```diff
@@ -0,0 +1,32 @@
+import { IO, AOProcess, ArweaveSigner, createAoSigner } from '@ar.io/sdk';
+import Arweave from 'arweave';
+
+const registryId = '6_b3wHAtM4WLUyrVJmfCPocRY3glayaEqZz8YqumcWQ';
+
+async function main() {
+  const arweave = Arweave.init();
+  const jwk = await arweave.wallets.generate();
+  const signer = createAoSigner(new ArweaveSigner(jwk));
+  const ario = IO.init();
+  const arnsRecords = await ario.getArNSRecords({
+    limit: 50_000,
+  });
+  const antIds = arnsRecords.items.map((record) => record.processId);
+
+  const registry = new AOProcess({ processId: registryId });
+  let count = 0;
+
+  for (const antId of antIds) {
+    console.log(`Registering ${count++}/${antIds.length}...`);
+    await registry.send({
+      tags: [
+        { name: 'Action', value: 'Register' },
+        { name: 'Process-Id', value: antId },
+      ],
+      signer,
+    });
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+  }
```

Comment:
batching here may make this more robust to timeouts - i.e. do 10 at a time and keep track of the last one you've submitted so you can start anew without having to spam more registers on retries

Reply:
RE slack decided to stick with 1 by 1 for now